run.go

```diff
@@ -491,25 +483,46 @@ func runSetupVolumeMounts(mountLabel string, volumeMounts []string) ([]specs.Mou
 			options = append(options, "rw")
 		}
 		if foundz {
-			if err := label.Relabel(spliti[0], mountLabel, true); err != nil {
-				return nil, errors.Wrapf(err, "relabeling %q failed", spliti[0])
+			if err := label.Relabel(host, mountLabel, true); err != nil {
+				return specs.Mount{}, errors.Wrapf(err, "relabeling %q failed", host)
 			}
 		}
 		if foundZ {
-			if err := label.Relabel(spliti[0], mountLabel, false); err != nil {
-				return nil, errors.Wrapf(err, "relabeling %q failed", spliti[0])
+			if err := label.Relabel(host, mountLabel, false); err != nil {
+				return specs.Mount{}, errors.Wrapf(err, "relabeling %q failed", host)
 			}
 		}
 		if rootProp == "" {
 			options = append(options, "private")
 		}
-
-		mounts = append(mounts, specs.Mount{
-			Destination: spliti[1],
+		return specs.Mount{
+			Destination: container,
 			Type:        "bind",
-			Source:      spliti[0],
+			Source:      host,
 			Options:     options,
-		})
+		}, nil
+	}
+	// Bind mount volumes specified for this particular Run() invocation
+	for _, i := range optionMounts {
+		mount, err := parseMount(i.Source, i.Destination, append(i.Options, "rbind"))
```

Comment:
You swithched bind above to rbind below, is that intentional?

Reply:
Yes, it brings the behavior for the CLI flag closer to volumes that were already set up using this code path.  I always thought it was weird that we had to pass that as an option from `runCmd()`.